apps/server/src/modules/class/entity/testing/factory/class.entity.factory.ts

```diff
@@ -21,7 +21,7 @@ export const classEntityFactory = ClassEntityFactory.define(ClassEntity, ({ sequ
 		teacherIds: [new ObjectId(), new ObjectId()],
 		invitationLink: `link-${sequence}`,
```

Comment:
While we're at it: Maybe also a good idea to put something looking more like a URL, e.g. "`https://www.example.com/invite/${sequence}`". Not really important though...

Reply:
noted. However, we have nothing regadring url format. For me, this is a good indication that we should not expect url format actually.